### PR TITLE
Upgrade `idb-storage` dependencies

### PR DESCRIPTION
**Pull request by devashishdxt**

### CODE REVIEW

File: storages/idb-storage/src/lib.rs [hunk 1]
```diff
@@ -182,19 +190,20 @@ impl Store for IdbStorage {
...
             None => {
                 return Ok(Box::pin(empty()));
             }
         };
 
         let mut rows = Vec::new();
-        let mut current_key = cursor.key().err_into()?;
-        let mut current_row = cursor.value().err_into()?;
+        let mut current_key = cursor.key().err_into()?.unwrap_or_else(|| JsValue::NULL);
+        let mut current_row = cursor.value().err_into()?.unwrap_or_else(|| JsValue::NULL);
```
Comment (ever0de) [inline storages/idb-storage/src/lib.rs:206]:
```suggestion
        let mut current_key = cursor.key().err_into()?.unwrap_or(JsValue::NULL);
        let mut current_row = cursor.value().err_into()?.unwrap_or(JsValue::NULL);
```

File: storages/idb-storage/src/lib.rs [hunk 2]
```diff
@@ -205,11 +214,11 @@ impl Store for IdbStorage {
             rows.push((key, row));
 
             cursor.advance(1).await.err_into()?;
-            current_key = cursor.key().err_into()?;
-            current_row = cursor.value().err_into()?;
+            current_key = cursor.key().err_into()?.unwrap_or_else(|| JsValue::NULL);
+            current_row = cursor.value().err_into()?.unwrap_or_else(|| JsValue::NULL);
```
Comment (ever0de) [inline storages/idb-storage/src/lib.rs:218]:
```suggestion
            current_key = cursor.key().err_into()?.unwrap_or(JsValue::NULL);
            current_row = cursor.value().err_into()?.unwrap_or(JsValue::NULL);
```

**Review by ever0de — COMMENTED** [review]
can you fix the clippy issues?

**Review by ever0de — APPROVED** [review]

**Review by panarch — APPROVED** [review]
:D thanks a lot for the contribution!

**Pull request merged**